src/Compiler/Compiler.cpp

```diff
@@ -700,6 +708,12 @@ bool Compiler::RegisterCommands(TclInterpreter* interp, bool batchMode) {
         compiler->ErrorMessage(out.str());
         return TCL_ERROR;
       }
+    } else {
```

Comment:
this is not working. why it was changed?

Reply:
Constraint file was being sourced twice. One in this method, another time in the project method. 
Especially obvious when multiple constraint files are used.
What is not working exactly? It is working fine with all the tests I did with multiple constraint files.

Reply:
This code is not reachable since compiler->m_tclCmdIntegration always true